src/utils/strings.js

```diff
@@ -58,3 +58,21 @@ export function findUrls(str) {
     }
   })
 }
+
+/**
+ * Convert string to finite float.
```

Comment:
I think this is a little more specific.  Here's an updated API.   Please also add unit tests for these and some boundary conditions with NaN, etc.

```
/**
 * Convert a string-encoded float to a truncated float, of fixed-length `len` or no decimal point expansion
 * if equal to zero.  e.g. with `len=3`:
 *  - '12.34567' -> 12.345
 *  - '12.340' -> 12.34 // same with '12.34'
 *  - '12.300' -> 12.3   // same with '12.3'
 *  - '12.000' -> 12 
 */

Reply:
i have done.